job_runner/jobs/email_scan/indeed_utils/index.js

```diff
@@ -29,14 +31,23 @@ export const scanIndeedEmail = (rawEmailString = '', index) => {
   }
 
   const indexOfDocType = rawEmailString.indexOf(DOC_TYPE_HTML);
+  const indexOfHTML = rawEmailString.indexOf('<html');
 
+  let htmlStartIndex = indexOfDocType;
   if (indexOfDocType < 0) {
-    const msg = `WARNING: indeed email did not start with ${DOC_TYPE_HTML} or has no html`;
-    console.log(msg);
+    htmlStartIndex = indexOfHTML;
+  }
+  if (htmlStartIndex < 0) {
+    const msg = `WARNING: indeed email has no html and is not recognized`;
```

Comment:
is the pre-commit hook not running for you? it should run linter & fix automatically anything it can.
You need to be on node 10 at least, and if npm v7, it has to be at least 7.1.2 (and git >= 2.13)

Reply:
i did `--no-verify` shhhh...

Reply:
Oh, so you like pain 👀

Reply:
i'm still poking at these features, i will be fixing the lint errors before merge